### PR TITLE
Remove the 🥕 since we have modes in the go to file

### LINKED ISSUE #163957: Remove `?` dropdown from command center
Given that we now have this option in the Command Center..

CleanShot 2022-10-18 at 10 54 58@2x

I think it makes sense to remove this arrow button. It's misleading as it looks like the affordance to open the command center (i.e. what would open the new "home" state), not provide a shortcut to the `?` menu.

CleanShot 2022-10-18 at 10 54 51@2x

cc @jrieken

### PR DESCRIPTION
**Pull request by TylerLeonhardt**
Fixes #163957

### CODE REVIEW

File: src/vs/workbench/browser/layout.ts [hunk 1]
```diff
@@ -167,7 +167,11 @@ export abstract class Layout extends Disposable implements IWorkbenchLayoutServi
 		}
 		if (this.isVisible(Parts.TITLEBAR_PART)) {
 			top += this.getPart(Parts.TITLEBAR_PART).maximumHeight;
-			quickPickTop = this.titleService.isCommandCenterVisible ? quickPickTop : top;
+			quickPickTop = top;
+		}
+		// If the command center is visible then the quickinput should go over the title bar and the banner
+		if (this.titleService.isCommandCenterVisible) {
+			quickPickTop = 0;
```
Comment (jrieken) [inline src/vs/workbench/browser/layout.ts:174]: Unrelated change? Will also overlay the banner which I believe is unwanted?

**Review by jrieken — CHANGES_REQUESTED** [review]

File: src/vs/workbench/browser/layout.ts (hunk 1 above)
Comment (TylerLeonhardt) [reply src/vs/workbench/browser/layout.ts:174]: @jrieken I actually wanted that. because otherwise the command center disappears and it looks weird.

Comment (jrieken) [reply src/vs/workbench/browser/layout.ts:174]: makes sense to my but maybe /cc @daviddossett given emotion with the banner were high. I agree for better looks but I believe the point of the banner is to not be permanent?

Comment (daviddossett) [reply src/vs/workbench/browser/layout.ts:174]: Not sure I follow—what is the effect of this?

Comment (TylerLeonhardt) [reply src/vs/workbench/browser/layout.ts:174]: @daviddossett so you know we have this banner:
image

today, if you have that banner (be in an untrusted workspace) and you launch the command center this happens:
image
Actually I think that's a bug because really what the code was intending to do, originally, was have the quick pick be _under_ the banner so it didn't cover it basically with the idea that "overlaying anything over the banner is bad"... 

My change says "covering the banner for the quick pick is ok" and has the quick pick do what it does today in an untrusted workspace.

Comment (daviddossett) [reply src/vs/workbench/browser/layout.ts:174]: I think covering it is totally ok personally. It would be strange to decouple the command center button/input from the quick pick itself.

I don't actually see the titlebar element in the second screenshot. Is that intentional?

Comment (TylerLeonhardt) [reply src/vs/workbench/browser/layout.ts:174]: Yeah it is hidden when you click on it since it is often longer than the quickpick itself.

Comment (TylerLeonhardt) [reply src/vs/workbench/browser/layout.ts:174]: I think we're all in agreement. I'll defer to @jrieken for a re-review

**Review by jrieken — APPROVED** [review]
